### PR TITLE
unpack push_transaction

### PR DESCRIPTION
**Pull request by AlexeyUsachev**
i need some information about response method "push transaction". But I don't know how to unpack "ActionTraces". Unfortunately I didn't find description about response in documentation. 
p.s. actionTraces isn't important for me.

### CODE REVIEW

File: responses.go [hunk 1]
```diff
@@ -289,15 +289,27 @@ type PushTransactionFullResp struct {
...
+	ID              Checksum256      `json:"id"`
+	BlockNum        uint32           `json:"block_num"`
+	BlockTime       JSONTime         `json:"block_time"`
+	ProducerBlockId string           `json:"producer_block_id"`
+	Receipt         ProcessedReceipt `json:"receipt"`
+	Elapsed         uint64           `json:"elapsed"`
+	NetUsage        uint64           `json:"net_usage"`
+	scheduled       bool             `json:"scheduled"`
+	// except
+	ActionTraces         []Trace  `json:"action_traces"`         // it isn`t unpack! Who knows where it has been described in the documentation?
```
[the remark this answers is not in the document]
Comment (maoueh) [reply responses.go]: @AlexeyUsachev Can you give more details about this line exactly?

[the remark this answers is not in the document]
Comment (AlexeyUsachev) [reply responses.go]: I am sorry. Unfortunately no. Maybe I can learn it next weekend.

also It didn't work on previous version :-(.

[the remark this answers is not in the document]
Comment (AlexeyUsachev) [reply responses.go]: I fix it

**Comment by abourget** [comment]
Do you still need this? Those are breaking changes.. any other way? Or perhaps the lib has changed already to satisfy your needs?

**Pull request closed without merging**